tests/upgrades/test_yum_plugins.py

```diff
@@ -161,13 +174,47 @@ def _host_location_update(self, client_container_name=None, loc=None):
 
     def _publish_content_view(self, org, repolist):
         """publish content view and return content view"""
+
         content_view = entities.ContentView(organization=org).create()
         content_view.repository = repolist
         content_view = content_view.update(['repository'])
-        content_view.publish()
+        call_entity_method_with_timeout(content_view.publish, timeout=3400)
         content_view = content_view.read()
         return content_view
 
+    def _create_rh_rhel_tools_repos(self, org_id):
+        """ Upload manifest, enable RHEL7 and tools repos
+        and sync them
+
+        :param: int org_id: Organization ID
+
+        :return: nailgun.entities.Repository: repository
+        """
+
+        assert org_id, 'org_id has to be int'
```

Comment:
I did not understand this assertion, why this required? If later org_id changed to alphanumeric unique string then the whole test will not run.  Is there any dependency that organisation should be int ?

Reply:
`org.id` always remain `int` and idea to keep this check to avoid pass Id as string .

Reply:
why we should have assertion on org_id in upgrade test at all? org.id is a system-generated unique value which currently int, but it can also be a string. The question here is why that assertion is necessary to fail the test?

Reply:
it's removed now (as I was thinking to use as a helper in upgrade but will check later on that)... also org.id never be str hence removed the assertion.